src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/Resources/UpdateResourcesValidation.feature.disabled

```diff
@@ -0,0 +1,223 @@
+                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "shortDescription": "Sick Leave"
+                    }
+                  """
+             Then it should respond with 204
+             When a GET request is made to "/ed-fi/absenceEventCategoryDescriptors/{id}"
+             Then it should respond with 200
+              And the response body is
+                  """
+                    {
+                        "codeValue": "Sick Leave",
+                        "description": "Sick Leave Edited",
+                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "shortDescription": "Sick Leave"
+                    }
+                  """
+
+        @ignore
+        Scenario: Verify error handling updating a resource with invalid data
+            # The id value should be replaced with the resource created in the Background section
+             When a PUT request is made to "/ed-fi/absenceEventCategoryDescriptors/<id>" with
+                  """
+                    {
+                        "codeValue": "Sick Leave",
+                        "description": "Sick Leave Edited",
+                        "namespace": "AbsenceEventCategoryDescriptor",
+                        "shortDescription": "Sick Leave"
+                    }
+                  """
+             Then it should respond with 400
+              And the response body is
+                  """
+                    {
+                        "detail": "Identifying values for the AbsenceEventCategoryDescriptor resource cannot be changed. Delete and recreate the resource item instead.",
+                        "type": "urn:ed-fi:api:bad-request:data",
+                        "title": "Data Validation Failed",
+                        "status": 400,
+                        "correlationId": null
+                    }
+                  """
+
+        @ignore
+        Scenario: Verify that response contains the updated resource ID and data
+            # The id value should be replaced with the resource created in the Background section
+             When a PUT request is made to "/ed-fi/absenceEventCategoryDescriptors/<id>" with
+                  """
+                    {
+                        "codeValue": "Sick Leave",
+                        "description": "Sick Leave Edited",
+                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "shortDescription": "Sick Leave"
+                    }
+                  """
+             Then it should respond with 204
+                  | location |  |
+              And the response headers includes
+              #replace header <id> with the correct value
+                  """
+                    {
+                        "location": "ed-fi/absenceEventCategoryDescriptors/<id>",
+                    }
+                  """
+
+        @ignore
```

Comment:
These two validation scenarios may be covered already in https://github.com/Ed-Fi-Alliance-OSS/Data-Management-Service/blob/main/src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/ValidationErrors.feature.

If not, I wonder if it would make sense to move them to that other file? Alternately, we could take the scenarios from `ValidationErrors.feature` and move them into these four "CRUD" feature files.

Reply:
I think we can review them and see how many of them can be combined, The Scenario's names from Resources folder are more generic.

Reply:
Do you want to review that in this ticket or at another time?

Reply:
I would prefer to handle this in a separate ticket. [DMS-182](https://tracker.ed-fi.org/browse/DMS-182)